bin/yb-ctl

```diff
@@ -1856,9 +1849,13 @@ class ClusterControl:
         initdb_binary_path = self.options.get_binary_path("initdb")
 
         logging.info("Running initdb to initialize YSQL metadata in the YugaByte DB cluster.")
-        with SetAndRestoreEnv(
-            {'FLAGS_pggate_master_addresses': self.options.master_addresses,
-             'YB_ENABLED_IN_POSTGRES': '1'}):
+        env = {'FLAGS_pggate_master_addresses': self.options.master_addresses,
+               'YB_ENABLED_IN_POSTGRES': '1'}
+        certs_dir = self.get_certs_dir()
+        if certs_dir:
+            env.update({'FLAGS_use_client_to_server_encryption': '1',
+                        'FLAGS_certs_dir': certs_dir})
+        with SetAndRestoreEnv(env):
```

Comment:
I am surprised we're still passing flags to pggate through environment variables. I thought that was the responsibility of yb-tserver. I would certainly try to avoid expanding the set of flags we're passing that way.

Reply:
You are right, `yb-tserver` propagates all the flag to postgres process when it starts it. But in this particular case `yb-tserver` doesn't participate in this scenario. `yb-ctl` starts `initdb` (i.e. postgres process) on its own. So it should provide all flags to it via env variables